docker_files/etc/confd/templates/config.json.tmpl

```diff
@@ -0,0 +1,142 @@
+{
+	"wsPort": {{getv "/lisk/wsport" "8001"}},
+	"httpPort": {{getv "/lisk/httpport" "8000"}},
+	"address": "{{getv "/lisk/address" "0.0.0.0"}}",
+	"version": "{{getenv "LISK_VERSION"}}",
+	"minVersion": "{{getenv "LISK_MIN_VERSION"}}",
+	"fileLogLevel": "{{getv "/lisk/fileloglevel" "info"}}",
+	"logFileName": "{{getv "/lisk/logfilename" "logs/lisk.log"}}",
+	"consoleLogLevel": "{{getv "/lisk/consoleloglevel" "error"}}",
+	"trustProxy": {{getv "/lisk/trustproxy" "false"}},
+	"topAccounts": {{getv "/lisk/topaccounts" "false"}},
+	"cacheEnabled": {{getv "/lisk/cacheenabled" "false"}},
+	"wsWorkers": {{getv "/lisk/wsworkers" "1"}},
+	"db": {
+		"host": "{{getv "/lisk/db/host" "localhost"}}",
+		"port": {{getv "/lisk/db/port" "5432"}},
+		"database": "{{getv "/lisk/db/database" "lisk_main"}}",
+		"user": "{{getv "/lisk/db/user" "lisk"}}",
+		"password": "{{getv "/lisk/db/password" "password"}}",
+		"min": {{getv "/lisk/db/min" "10"}},
+		"max": {{getv "/lisk/db/max" "95"}},
+		"poolIdleTimeout": {{getv "/lisk/db/poolidletimeout" "30000"}},
+		"reapIntervalMillis": {{getv "/lisk/db/reapintervalmillis" "1000"}},
+		"logEvents": ["error"],
+		"logFileName": "{{getv "/lisk/db/logfilename" "logs/lisk_db.log"}}"
+	},
+	"redis": {
+		"host": "{{getv "/lisk/redis/host" "127.0.0.1"}}",
+		"port": {{getv "/lisk/redis/port" "6380"}},
+		"db": {{getv "/lisk/redis/db" "0"}},
+		"password": {{getv "/lisk/redis/password" "null"}}
```

Comment:
I believe here you want to set `null` instead of `"null"`, consider `null` without quote means no password. But if you quote it will check for that password.

Reply:
Indeed, good catch.

Reply:
This will output `null` not `"null"`: notice there are not double quotes around the expression in double curly braces (compare to `host` for instance)